rosdep/python.yaml

```diff
@@ -5880,6 +5880,22 @@ python3-aiohttp-cors:
   fedora: [python-aiohttp-cors]
   gentoo: [dev-python/aiohttp-cors]
   ubuntu: [python3-aiohttp-cors]
+python3-aiortc-pip:
+  debian:
+    pip:
+      packages: [aiortc]
+  fedora:
+    pip:
+      packages: [aiortc]
+  gentoo:
+    pip:
+      packages: [aiortc]
+  osx:
+    pip:
+      packages: [aiortc]
+  ubuntu:
+    pip:
+      packages: [aiortc]
```

Comment:
@Interpause I see `python3-aiortc` available in several platform specific package managers (e.g. Debian, Ubuntu).

Consider adding a `python3-aiortc` key referring those wherever applicable.

Reply:
We generally prefer system packages over pip : https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md